src/ServicePulse.Host/app/js/app.constants.js

```diff
@@ -5,7 +5,7 @@
         .constant('showPendingRetry', false)
         .constant('scConfig', {
             service_control_url: 'http://localhost:33333/api/',
-            monitoring_url: 'http://localhost:1234/metrics/'
+            monitoring_urls: ['http://localhost:1234/metrics/', 'http://localhost:1235/metrics', 'http://localhost:1236/metrics']
```

Comment:
What I am saying is that you configured it to hold, 3 values. I am asking if we should push it with those 3 values. As every dev environment will use those values.